# Updated by fix_deps.py
../newlib/Debug/communication_exception_x86_64.o: \
 communication_exception.cc communication_exception.h
:
communication_exception.cc:
communication_exception.h:
